### ISSUE TITLE
Updated analysis: fusion-summary with v15 fusion files and downstream

### ISSUE THREAD

**Issue by jaclyn-taroni** [AlexsLemonade/OpenPBTA-analysis#578]
#### What analysis module should be updated and why?

`fusion-summary` creates files of fusions of interest from the `pbta-fusion-putative-oncogenic.tsv` file, which is being updated in v15 (#569). The `molecular-subtyping-embryonal` and `molecular-subtyping-EPN` are downstream of the `fusion-summary` module.

#### What changes need to be made? Please provide enough detail for another participant to make the update.

The `01-fusion-summary.Rmd` notebook in`fusion-summary` needs to be updated. It is insufficient to rerun it because new processing criteria for the fusions was used and now it appears, at least in the case of the embryonal tumor fusions of interest, there are new instances of fusions that previously need to be filled in as all `0`:

https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/286ff25022930024bb9812e3cfad5410a2cf49c8/analyses/fusion-summary/01-fusion-summary.Rmd#L159

So these chunks

https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/286ff25022930024bb9812e3cfad5410a2cf49c8/analyses/fusion-summary/01-fusion-summary.Rmd#L153

https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/286ff25022930024bb9812e3cfad5410a2cf49c8/analyses/fusion-summary/01-fusion-summary.Rmd#L168

Need to be looked at carefully to see what fusions of interest need to be filled in with all `0`. Notably, the TSV results of module then needs to go on deck for v16 release, so let's plan to get a `planned data release` issue filed at the same time a PR for this update gets filed!

Then the downstream subtyping modules (`molecular-subtyping-embryonal` and `molecular-subtyping-EPN`) then need to be rerun and checked for any change in interpretation.

#### What input data should be used? Which data were used in the version being updated?

`pbta-fusion-putative-oncogenic.tsv`

#### When do you expect the revised analysis will be completed?

I think this should be completed around the same time as #574. ~1 day barring any unforeseen complications.

#### Who will complete the updated analysis?

Likely someone from the CCDL.

**Comment by jaclyn-taroni** [comment]
We also have to keep an eye on the logic for CI in `01-fusion-summary.Rmd`. We may need to take an approach where we enrich for certain fusions in the subset files for CI or include the full `pbta-fusion-putative-oncogenic.tsv`.

**Comment by jaclyn-taroni** [comment]
During #605, I realized that we were not using the fusion summary files included in the data download and fixed that. Before I fixed it, I reran both modules and found that some specimens we had classified as CNS NB-FOXR2 now have _FOXR2_ fusions (_FOXR2_ was overexpressed). We have not yet gotten to the logic/classification of the EPN samples. We are planning on looking at that next week (https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/555#issuecomment-594244746). If v16 (#601) doesn't go out before Monday, we should plan to use the fusion summary files committed to the repository.
